backend/src/apiserver/resource/resource_manager.go

```diff
@@ -1792,12 +1795,3 @@ func (r *ResourceManager) GetTask(taskId string) (*model.Task, error) {
 	}
 	return task, nil
 }
-
-// Fetches run metric entries for a given run id.
-func (r *ResourceManager) GetRunMetrics(runId string) ([]*model.RunMetric, error) {
-	metrics, err := r.runStore.GetMetrics(runId)
```

Comment:
If we are removing the `GetRunMetrics()` method, looks like this is also the only use case for `GetMetrics()` in run storage. Shall we remove this method too?

Reply:
Created a separate PR and added @Linchin's suggestion there: https://github.com/kubeflow/pipelines/pull/9391